files/zh-cn/mdn/community/contributing/translated_content/index.md

```diff
@@ -21,8 +21,9 @@ original_slug: MDN/Contribute/Localize
 
 ### 汉语（zh-CN，zh-TW）
 
-- 讨论组：[Telegram (MozTW L10n channel)](https://moztw.org/tg)
-- 目前的志愿者：[Irvin](https://github.com/irvin)、[t7yang](https://github.com/t7yang)、[yin1999](https://github.com/yin1999)
+- Moz://TW 讨论组：[Telegram (MozTW L10n channel)](https://moztw.org/tg)
+- Mozilla China 讨论组：[Mozilla China](https://t.me/mozilla_china)
+- 目前的志愿者：[Irvin](https://github.com/irvin)、[t7yang](https://github.com/t7yang)、[yin1999](https://github.com/yin1999)、[LaoshuBaby](https://github.com/LaoshuBaby)
```

Comment:
This is used to list l10n team members, so please revert this change

![image](https://user-images.githubusercontent.com/15844309/209038666-fd2ea9eb-6a02-4b24-aad0-caf90a31a06f.png)

Reply:
Thanks for pointing out that if there is a clear source for this, it must be according to the source. 
I added myself before because I want to help establish communication when noticed someone said that they can't get in touch with others.